Jbarget/oxymore
# So/eye

### What changes have you made?
A couple of UX improvements: 
- added a `hideCountryData` function so that the user can hide the country data by clicking anywhere on the page 
- the Country Data can also be hidden on selecting a new Data Set from the Item Picker
- added a keyboard event so that the user can close the Country Data using the escape key 
- set the initial state of the `SelectedDataSet` to the middle item of the Item Picker 
- created a condition to display the data with the digit group separator for Spanish or English i.e. comma or decimal 
- added a bounce to the Country Data to reinforce the feeling of floating in the clouds
- created translations for Data Sets and Country Data 

New features: 
- Podcast with play and pause functionality 

New components:
- Country Data has been abstracted out into a component called `CountryData` to make the  `Eye` code easier to read

### Which issue(s) does this PR fix?

Fixes #379 
Fixes #301 
Fixes #372 
Fixes #370 
Fixes #371 

### Screenshots (if there are design changes)
<img width="1438" alt="Screenshot 2021-02-10 at 21 31 55" src="https://user-images.githubusercontent.com/53219789/107575141-6faa9600-6be7-11eb-947a-ee53be29e379.png">


### How to test
- Check that the audio plays on click of the play button 
- Check that the audio pauses on click of the pause button 
- Click on a random eye to display its Country Data and click anywhere on the page to close 
- Select a new Data Set from the Item Picker, click on an eye to open its Country Data, select a new Data Set and see that the Country Data is hidden

### Known issues
- The country names do not have translation functionality
